mizar/networkpolicy/networkpolicy_util.py

```diff
@@ -69,14 +72,19 @@ def handle_networkpolicy_create_update(self, name, pod_label_dict, policy_types)
 
                 ep.set_data_for_networkpolicy(data_for_networkpolicy)
                 ep.update_networkpolicy_per_endpoint(data_for_networkpolicy)
+                for label in data_for_networkpolicy["ingress"]["label_networkpolicies_map"]:
+                    networkpolicy_opr.store.add_label_networkpolicy_ingress(label, data_for_networkpolicy["ingress"]["label_networkpolicies_map"][label])
+                for label in data_for_networkpolicy["egress"]["label_networkpolicies_map"]:
```

Comment:
This can be converted into network policy object specs

Reply:
data_for_networkpolicy is not per networkpolicy. It's per endpoint. It is to calculate each endpoint's networkpolicy related data.